auditlog/middleware.py

```diff
@@ -1,97 +1,31 @@
-import threading
-import time
-from functools import partial
+import contextlib
 
-from django.apps import apps
-from django.conf import settings
-from django.db.models.signals import pre_save
-from django.utils.deprecation import MiddlewareMixin
+from auditlog.context import set_actor
 
-from auditlog.models import LogEntry
 
-threadlocal = threading.local()
-
-
-class AuditlogMiddleware(MiddlewareMixin):
+class AuditlogMiddleware(object):
```

Comment:
```suggestion
class AuditlogMiddleware:
```

Reply:
Oh my, this is a pretty ancient line of code!

Reply:
Fixed.